optional/config/tech-docs.yml.tt

```diff
@@ -21,15 +21,15 @@ enable_search: false
 ga_tracking_id:
 
 # Enable multipage navigation in the sidebar
-multipage_nav: false
+multipage_nav: true
 
 # Enable collapsible navigation in the sidebar
 collapsible_nav: false
```

Comment:
Is there any reason not to use collapsible navigation?

Right now it looks a bit weird (in my opinion), with the headings for the support links trailing at the end:

![Screenshot 2022-03-29 at 13 39 25](https://user-images.githubusercontent.com/503614/160613062-31ccd0ec-d62c-4ebc-bd96-a1126b9facb2.png)

Reply:
Ah yes good call! I'll fix that.